### PR TITLE
refactor: remove functions_concepts_error_object

### PR DESCRIPTION
**Pull request by grant**
Removes unused and not useful sample.

### CODE REVIEW

**Comment by grant** [comment]
@ace-n Can you review this assuming the tests pass?

File: functions/concepts/test/index.test.js [hunk 1]
```diff
@@ -15,21 +15,7 @@
 
 'use strict';
 
-const sinon = require('sinon');
-const assert = require('assert');
 const tools = require('@google-cloud/nodejs-repo-tools');
```
Comment (ace-n) [inline functions/concepts/test/index.test.js]: Can we remove this file, or should we add a (simple) test for `executionCount`?

Comment (grant) [reply functions/concepts/test/index.test.js]: The goal of this PR is not to remove this test file, just to remove the `functions_concepts_error_object` region tag.

I'm not sure why the other functions in this file don't have tests, but that's a separate issue.

**Comment by grant** [comment]
I can delete the test file upon request, but I think it's fine to leave it blank for the other functions that should be tested.

**Comment by grant** [comment]
@ace-n WDYT about this PR?

**Comment by ace-n** [comment]
Apologies for the delay. I'd say go ahead and delete the test file.

**Comment by grant** [comment]
> Pardon the noise (bad push).

Removed the test file. PTAL.

**Comment by grant** [comment]
Can be merged when tests pass.

**Comment by grant** [comment]
Not sure why the PR is out of date w.r.t. master twice.

**Comment by grant** [comment]
@ace-n Can you look to why the `functions/concepts` test is failing? I really think there is a bug with the testing infrastructure rather than this PR.

**Comment by ace-n** [comment]
This error was a result of deleting a test file and not removing the
related Kokoro config files. Arguably, this test failure ("no tests found",
if you look at the logs) should be expected.

You'll have to remove the *functions/concepts* test configs a) internally
(from Kokoro), and b) from the *nodejs-docs-samples/.kokoro* folder. LMK if
you need more specific info on *where* all that cruft is. (I can probably
dig up an example CL.)

**Comment by grant** [comment]
Well, this is one reason to not remove the file...

```
> nodejs-docs-samples-functions-concepts@0.0.1 test /tmpfs/src/github/nodejs-docs-samples/functions/concepts
> mocha test/*.test.js --timeout=20000

Error: No test files found: "test/*.test.js"
npm ERR! Test failed.  See above for more details.


[ID: 3319830] Build finished after 72 secs, exit value: 1
```

Can we just proceed to remove the sample/region tag `functions_concepts_error_object` without removing the file?

Do you want me to remove this file too?:
https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/master/.kokoro/functions/concepts.cfg

**Comment by fhinkel** [comment]
I'd prefer if we remove the empty test file.

**Comment by grant** [comment]
> You'll have to remove the *functions/concepts* test configs a) internally
(from Kokoro)
@ace-n @fhinkel Can you point me to a doc that guides me on how to locate and remove the test configs from Kokoro? I believe I have done so in the `.kokoro` folder already.

**Comment by fhinkel** [comment]
Our codelab explains it very well: go/kokoro-codelab

**Comment by grant** [comment]
Kokoro config removed with cl/276701784.

**Comment by grant** [comment]
Approved. Tests pass. Merging.

**Pull request merged**